client/src/components/profile/components/HeatMap.js

```diff
@@ -2,6 +2,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Helmet } from 'react-helmet';
 import CalendarHeatMap from 'react-calendar-heatmap';
+import ReactTooltip from 'react-tooltip';
+import addDays from 'date-fns/add_days';
+import addMonths from 'date-fns/add_months';
 import startOfMonth from 'date-fns/start_of_month';
```

Comment:
```suggestion
```
If you do go with my suggestion about the start, this can go.  Otherwise, just ignore this comment!